v2/app/api_framework.go

```diff
@@ -110,5 +112,14 @@ func configureAPIFramework(pc *parsectx.Context, services []*Service, res *parse
 		})
 	}
 
+	// Call modifySvcDesc for subscriptions and auth handlers, even though they don't actually
+	// modify the framework description, to actually tag the package as a service.
+	for _, ss := range subscriptions {
+		modifySvcDesc(ss.Package(), nil, func(svc *Service, desc *apiframework.ServiceDesc) {})
+	}
+	for _, ah := range authHandlers {
+		modifySvcDesc(ah.Package(), nil, func(svc *Service, desc *apiframework.ServiceDesc) {})
+	}
+
```

Comment:
For subscriptions I didn't do this on purpose, as a subscription does not create an API service, so there's no `apiframework.ServiceDesc` - but it still can create a service.

What was the need to do this here?